PixiEditor/ViewModels/ImportFilePopupViewModel.cs

```diff
@@ -132,12 +132,19 @@ private void CheckForPath(string path)
         {
             if (File.Exists(path) && (path.EndsWith(".png") || path.EndsWith(".jpeg") || path.EndsWith(".jpg")))
             {
-                PathButtonBorder = "#b8f080";
-                PathIsCorrect = true;
-                _filePath = path;
-                BitmapImage bitmap = new BitmapImage(new Uri(path));
-                ImportHeight = bitmap.PixelHeight;
-                ImportWidth = bitmap.PixelWidth;
+                try
+                {
+                    PathButtonBorder = "#b8f080";
+                    PathIsCorrect = true;
+                    filePath = path;
+                    BitmapImage bitmap = new BitmapImage(new Uri(path));

```

Comment:
After hex editing a image file it threw an `FileFormatException` with the message 'The image format is unrecognized.'